### PR TITLE
feat(test): add tests for Auth helper class

### PR DESCRIPTION
**Pull request by papidb**
- Test JWT token signing with default and custom options
  - Test JWT token verification with different secret configurations
  - Test random token generation with proper format validation
  - Test SHA256 token hashing with edge cases
  - Add integration tests for method combinations

### CODE REVIEW

File: apps/server/src/common/utils/authentication/auth.helper.spec.ts [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+          return 'mutated-secret';
+        case 'auth.jwtExpires':
+          return '2h';
+        default:
+          return null;
+      }
+    });
+
+    await auth.verify('token');
+    expect(jwtService.verifyAsync).toHaveBeenCalledWith('token', {
```
[the remark this answers is not in the document]
Comment (papidb) [reply apps/server/src/common/utils/authentication/auth.helper.spec.ts:65]: added.

**Review by norb-c — APPROVED** [review]
